plugins/src/lib.rs

```diff
@@ -431,7 +431,7 @@ impl<'a> ServiceGenerator<'a> {
                 }
 
                 async fn serve(self, ctx: tarpc::context::Context, req: #request_ident)
-                    -> Result<#response_ident, tarpc::ServerError> {
+                    -> ::std::result::Result<#response_ident, tarpc::ServerError> {
                     match req {
                         #(
                             #request_ident::#camel_case_idents{ #( #arg_pats ),* } => {
```

Comment:
what about the use of `Ok` and `Some` etc.? Do you want to clean those up, too?

Reply:
Sure, I just missed some of those, but it makes sense.